src/sentry/static/sentry/app/components/splitLayout.jsx

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+import SpreadLayout from './spreadLayout';
+
+const SplitLayout = ({children, className, responsive, ...props}) => {
```

Comment:
What is the difference between `SplitLayout` and `SpreadLayout`? Maybe add a comment?